apis/fluentbit/v1alpha2/plugins/output/prometheus_remote_write_types.go

```diff
@@ -0,0 +1,93 @@
+package output
+
+import (
+	"fmt"
+
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+	"github.com/fluent/fluent-operator/pkg/utils"
+)
+
+// +kubebuilder:object:generate:=true
+
+// An output plugin to submit Prometheus Metrics using the remote write protocol
+type PrometheusRemoteWrite struct {
+	// IP address or hostname of the target HTTP Server, default: 127.0.0.1
+	Host string `json:"host"`
+	// Basic Auth Username
+	HTTPUser *plugins.Secret `json:"httpUser,omitempty"`
+	// Basic Auth Password.
+	// Requires HTTP_user to be se
+	HTTPPasswd *plugins.Secret `json:"httpPassword,omitempty"`
```

Comment:
```suggestion
	HTTPPasswd *plugins.Secret `json:"httpPasswd,omitempty"`
```
should be consistent with `http_passwd` https://docs.fluentbit.io/manual/pipeline/outputs/prometheus-remote-write